### ISSUE TITLE
onProgress not called after first video (Android)

### ISSUE THREAD

**Issue by zachrnolan** [TheWidlarzGroup/react-native-video#543]
Our app plays multiple videos in a row. The video component is re-rendered each time with a new video uri. Everything works as expected on iOS, but we are having issues on Android:

First Video: onLoadStart => onLoad => onProgress => onEnd
Second Video: onLoadStart => onLoad, then nothing happens. The video plays, but we do not get any logs for onProgress, but we do get one for onEnd.

Any ideas why this is happening on Android?

RN: 0.42.0
RN Video: 1.0.0

**Comment by oppositeminddev** [comment]
Did you ever find a solution to this? I'm experiencing the same issue, on android only

**Comment by lequanghuylc** [comment]
Same issue here. Did anyone solve this?

**Comment by zakster12** [comment]
Same issue here. Only on android. @zachrnolan did you find any solution to this?

**Comment by artushin** [comment]
There's a conditional (`if (mMediaPlayerValid && !isCompleted &&!mPaused)`) which controls whether the onProgress event is fired. When you play another video, isCompleted isn't being reset. You can force it to be reset by calling `player.seek(0)` on the onLoad callback. 

Maintainers, does setting `isCompleted` to false where you emit `EVENT_LOAD_START` sound reasonable? If so, I can put in a PR.

**Comment by fsguru007** [comment]
Same issue here. I added `player.seek(0)` in onLoad callback but it does not solve the issue.

**Comment by nergi-r** [comment]
@artushin I've tried adding `isCompleted` after emitting `EVENT_LOAD_START`, but it still didn't fix the problem.
In the end, I used javascript setTimeout to simulate onprogress event. In case anyone interested, I'll just paste my workaround here. Still hope that this issue will be addressed officially.
```
  onLoad() {
    this.props.updateCurrentTime(0);
    let interval = 250;
    let currentTimer = setInterval(() => this.props.updateCurrentTime(this.props.currentTime + interval), interval);
    this.setState({currentTimer: currentTimer});
  }
  onStop() { clearInterval(this.state.currentTimer); }
```

**Comment by fsguru007** [comment]
That's an interesting way. In my case I solved the issue by using `react-native-video/android-exoplayer` instead of `react-native-video/android`.

**Comment by vincent-cihan** [comment]
Same issue here.In my case I solved the issue by adding player.seek(0) in onEnd callback.

**Comment by cobarx** [comment]
This has been fixed by #778
